arbitrum-docs/run-arbitrum-node/nitro/01-build-nitro-locally.md

```diff
@@ -189,3 +196,17 @@ To run your node using the generated binaries, use the following command from th
 ```bash
 ./target/bin/nitro <node parameters>
 ```
+
+#### WASM module root error
```

Comment:
I think this section could be transformed into an admonition ("info", for example).
And we should let the reader know this is a temporary issue with v2.3.4 only, as we want to avoid users modifying the WASM Module root used by default, imo.

What do you think?